src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,31 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("completeOrder was called with userId: {}", userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
-        // NOTE: In production ready code this order identifier should be generated by DB
-        // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.info(
+                "comleteOrder was called and run successfully with userId: {}, orderId: {}",
+                userId, order.getOrderId());
+        System.out.println("completeorder");
```

Comment:
Replace the System.out.println statement with a proper logging statement or remove it if it is not necessary.